docs/gigabyte-rgb-fusion2-guide.md

```diff
@@ -86,3 +86,13 @@ Homebrew, along with a script to run on wake that issues the necessary
 liquidctl commands and restores desired lighting effects.
 
 [automate the configuration at boot time]: ../README.md#automation-and-running-at-boot
+
+## Fading between colors
+
+The `extra/contrib/fusion_rgb_cycle.py` script can be used to have one of these controllers cycle through a list of colors of the user's choice. It uses the `fixed` mode but updates the controller continuously. It can be invoked with:
+
+```
+# extra/contrib/fusion_rgb_cycle.py 350017 ff2608
+```
+
+If more two colors are specified, they will be cycled in turn. The [color space](https://facelessuser.github.io/coloraide/colors/) for cycling can be specified, eg `--space hsv`, channels can be specified with eg, `--channel led6` and the time in seconds for each color transition can be set with, eg `--speed 1.2`.
```

Comment:
```suggestion
If more than two colors are specified, they will be cycled in turn. The [color space] for cycling can be specified with `--space <space>`, channels can be specified with `--channel <channel>` and the time in seconds for each color transition can be set with `--speed <seconds>`.

[color space]: https://facelessuser.github.io/coloraide/colors/
```

Reply:
Did you mean to remove the hyperlink to the color space docs with this change? There are quite a lot of them, and they produce quite different visual effects, so I think those docs will help some people...

Reply:
No, I just used the split link `[link]` and link definition `[link]: url` form.  Mostly because I was already suggesting other adjustments to the paragraph, this document is doing hard line wraps, and a long URL can be a pain for those.

But elsewhere you've consistently used the normal `[link](url)` form, so you can simply ignore it.

---

That said, there's still a `than` missing at the start of the sentence.